numpy/core/multiarray.py

```diff
@@ -251,6 +251,44 @@ def concatenate(arrays, axis=None, out=None, *, dtype=None, casting=None):
     return arrays
 
 
+def concat(arrays, axis=None, out=None, *, dtype=None, casting=None):
+    """
+    concat((a1, a2, ...), axis=0, out=None, dtype=None, casting="same_kind")
+
+    Join a sequence of arrays along an existing axis.
+
+    Parameters
+    ----------
+    a1, a2, ... : sequence of array_like
+        The arrays must have the same shape, except in the dimension
+        corresponding to `axis` (the first, by default).
+    axis : int, optional
+        The axis along which the arrays will be joined.  If axis is None,
+        arrays are flattened before use.  Default is 0.
+    out : ndarray, optional
+        If provided, the destination to place the result. The shape must be
+        correct, matching that of what concatenate would have returned if no
+        out argument were specified.
+    dtype : str or dtype
+        If provided, the destination array will have this dtype. Cannot be
+        provided together with `out`.
+
+        .. versionadded:: 1.20.0
+
+    casting : {'no', 'equiv', 'safe', 'same_kind', 'unsafe'}, optional
+        Controls what kind of data casting may occur. Defaults to 'same_kind'.
+
+        .. versionadded:: 1.20.0
+
+    Returns
+    -------
+    res : ndarray
+        The concatenated array.
+    """
+    return concatenate(arrays, axis=None, out=None, dtype=None, casting=None)
+
```

Comment:
```suggestion
```
There's one too many blank lines.

Reply:
Sir, I am new to open source, would you please tell me how may I edit (remove those lines), do I have the option to edit the file right away? or do I need to make another PR?

Reply:
There should be a "commit suggestion" button that you can click. If it's not there, just do what @mattip said.